src/NHSD.BuyingCatalogue.Ordering.Contracts/NHSD.BuyingCatalogue.Ordering.Contracts.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DocumentationFile>D:\Project\GPITFutures\API\13236_ORDAPI\BuyingCatalogueOrdering\NHSD.BuyingCatalogue.Ordering.Contracts\NHSD.BuyingCatalogue.Ordering.Contracts.xml</DocumentationFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\NHSD.BuyingCatalogue.Ordering.Domain\NHSD.BuyingCatalogue.Ordering.Domain.csproj" />
+  </ItemGroup>
+
+</Project>
```

Comment:
We treat warning as errors in release builds so can you enable that for the release build? Also make sure the release build also has the element below.

```xml
<GenerateDocumentationFile>true</GenerateDocumentationFile>
```

Reply:
Also please make sure to install the latest pre-release version of StyleCop.

```powershell
Install-Package StyleCop.Analyzers -Version 1.2.0-beta.333
```

Reply:
Enabled warning in `release` builds.
Also installed `StyleCop.Analyzers`